src/layers/domain/core/fhir_transform.py

```diff
@@ -34,18 +31,10 @@ def create_fhir_model_from_product_team(product_team: ProductTeam, **kwargs) ->
         name=product_team.name,
         partOf=cpm_model.Reference(
             identifier=cpm_model.Identifier(
-                id=product_team.organisation.id,
-                value=product_team.organisation.name,
+                id=product_team.ods_code,
+                value=product_team.ods_code,
```

Comment:
bug??

Reply:
Ah, this happened in the rebase.  Both properties are mandatory and we've changed the way we use `Identifier` from NRLF.  In NRLF we used `system` and `value` as a set, whereas `id` and `value` are pretty close in meaning.  Need to discuss this one.